engine/battle/core.asm

```diff
@@ -6661,7 +6677,14 @@ BattleRandom:
 	ld a, [hl]
 	pop bc
 	pop hl
+	vc_hook fight_ret_c
```

Comment:
Any time there's a hook immediately followed by a patch, use only the `vc_patch`, and put an alias for the hook in the template.